src/types.ts

```diff
@@ -1800,7 +1800,7 @@ export interface ContractDeployRequest {
     assetId: string;
     vaultAccountId: string;
     constructorParameters?: object[];
```

Comment:
if you already remove the `getContractTemplateConstructor` endpoint, why not removing also the param?